3 data.sql

```diff
@@ -0,0 +1,21 @@
+insert guests(LastName, FirstName, HotelLocation, RoomType, FloorNumber, RoomNumber, CheckInDate, CheckOutDate)
```

Comment:
make sure this is running directly from the database

Reply:
fixed problem

Reply:
amazing!